packages/js/src/containers/TwitterEditor.js

```diff
@@ -2,6 +2,8 @@
 import { compose } from "@wordpress/compose";
 import { withDispatch, withSelect, dispatch as wpDataDispatch } from "@wordpress/data";
 import { validateTwitterImage } from "@yoast/helpers";
+import { TWITTER_IMAGE_SIZES } from "@yoast/social-metadata-previews/src/helpers/determineImageProperties";
```

Comment:
Let's not import an internal path directly.
If it is not exposed through the `index.js`, you could add it.